conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -601,11 +601,12 @@ private Iterable<MethodSpec> createSetters(
             Map<com.palantir.conjure.spec.TypeName, TypeDefinition> typesMap,
             boolean override,
             boolean strict) {
-        Collection<MethodSpec> setters = Lists.newArrayListWithExpectedSize(fields.size());
+        // Each field tends to fan out into many setters
```

Comment:
This is only true of container types (optional/set/list), not standard fields. Difficult to tell whether a 5x multiplier is ideal or not, but probably not worth spending much time optimizing